contrib/init/sysvinit-redhat/docker

```diff
@@ -117,6 +118,21 @@ check_for_cleanup() {
 	if [ -f ${pidfile} ]; then
 		/bin/ps -fp $(cat ${pidfile}) > /dev/null || rm ${pidfile}
 	fi
+
+        # /var/run and /run in sysvinit is not a tmpfs
+	for i in $libcontainerd/containerd/*; do
+	    if [ -d $i ]; then
+	       if [ ! -f $i/state.json ]; then
+		    stale=`echo $(basename $i)`
+		    rm -rf $i
+		    rm -rf $libcontainerd/$stale
+		    rm -rf /run/runc/$stale
+	       else
+		    # if restart don't unnecessary check all 
+                    break
+	       fi
+	    fi
+	done
```

Comment:
Looks like there's some mixed tabs/spaces here, and they make @tianon cry 😢 :wink:

<img width="488" alt="screen shot 2016-11-25 at 13 53 34" src="https://cloud.githubusercontent.com/assets/1804568/20625979/ba3924ae-b316-11e6-9f98-6d5728869741.png">